src/components/Select.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./Select.module.css";
+
+interface SelectProps<Value extends string> {
+  onChange: (value: Value) => void;
+  options: Value[];
+  selected: Value;
+}
+
+export function Select<Value extends string>({ onChange, options, selected }: SelectProps<Value>) {
```

Comment:
This is a little annoying to set up, which is why the favorites filter doesn't use a generic for select... but this is a bit more type safe! Tradeoffs.